packages/ra-core/src/controller/input/useReferenceArrayInputController.ts

```diff
@@ -127,20 +126,31 @@ const useReferenceArrayInputController = ({
         translate,
     });
 
-    const referenceBasePath = basePath.replace(resource, reference); // FIXME obviously very weak
     return {
         choices: dataStatus.choices,
         error: dataStatus.error,
         loaded,
         loading: dataStatus.waiting,
-        referenceBasePath,
```

Comment:
This is never used by child components. It was not passed in 2.0. I think it comes from a bad copy/paste of ReferenceArrayField during the hooks refactoring.